app-misc/hodie/hodie-1.5.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="a Latin date(1)"
+HOMEPAGE="http://hodie.sourceforge.net"
+SRC_URI="https://github.com/michiexile/${PN}/archive/${P}.tar.gz
+		https://github.com/nephros/hodie/commit/6f08fdd05b4624200ce39519716bca569976362b.patch -> ${P}-classic.patch"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+S="$WORKDIR"/${PN}-${P}
+
+PATCHES="${DISTDIR}"/${P}-classic.patch
```

Comment:
Array form here.

Reply:
> I am wondering the point of that -r1 ebuild. You can't have KEYWORDS defined in a pure git package, thats where the error comes from. But is that -r1 really necessary?

It's meant to pick one. Not sure which approach is better. to pull the "proper" 1.5 release tarballl and add a patch from git, or ust to put the latest commited version form git and pretend in the ebuild it's the 1.5 release.

Reply:
You can point to a commit between releases. Then name it like hodie-1.5_p20191118 (date when said commit was made). 
Example: 
https://gitweb.gentoo.org/repo/gentoo.git/tree/media-libs/rlottie/rlottie-0.0.1_pre20191101.ebuild
Check the "MYCOMMIT" and "SRC_URI" parts.

Reply:
> You can point to a commit between releases. Then name it like hodie-1.5_p20191118 (date when said commit was made).

 Thank you.  Adapted according to your example, and made the -1.5.ebuild to pull just the official released version.